### ISSUE TITLE
add visual cue when scrolling highlight into view

### ISSUE THREAD

**Issue by thiswillbeyourgithub** [burtonator/polar-bookshelf#1275]
**Is your feature request related to a problem? Please describe.**
When clicking "scroll into view" on a color highlight the sidebar scrolls automatically to show this highlight on top of the displayed list (top right corner).

But if it's among the last annotations of the document, it's going to scroll as much as it can and then stop. You then have to find the annotation visually through the list on display. On a high dpi screen it's annoying and could easily be made better by adding a visual cue like a blue aura or something.

What do you think ?

I hope you don't mind my frequent post on github, don't hesitate to tell me.

**Comment by burtonator** [comment]
Agreed... I think there's another bug for this but I'm not sure what the right visual cue would be ...

**Comment by thiswillbeyourgithub** [comment]
to me intuitively I was thinking : the borders of the annotation should blink/glow for 300ms in a blue color similar to the pagemark.
